app/services.py

```diff
@@ -0,0 +1,13 @@
+from time import time
+import random
+
+def integer_coefficients(amount=1, number_range=[1, 10], seed=time()):
+
+    random.seed(a=seed)
```

Comment:
I assume passing the seed in here means that the .choice and .randint will always return the same thing?

Reply:
correct